source/_components/homekit.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: "HomeKit"
+description: "Instructions how to setup the HomeKit component in Home Assistant."
+date: 2018-02-20 17:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: HomeKit
+ha_release: 0.64
+ha_iot_class:  "Local Polling"
```

Comment:
Is `Local Polling` the right class? HomeKit doesn't communicate with devices directly and it only works if Home Assistant and your Apple device are in the same local network.

Reply:
Or just remove the `iot class`?

Reply:
remove this key, it's not required